### DISCUSSION TITLE
so 2.3.30 so-steno fails to start

### DISCUSSION THREAD

**Discussion by danielbidwell** [Security-Onion-Solutions/securityonion#3460] [Unsupported Versions]
I am new to SecurityOnion.  I have a fresh install of 2.3.30 on ubuntu 18.04.5.  After everything comes up so-status shows that so-steno has an error.  docker logs so-steno shows errors of "Skipping generation of 'server_127.0.0.1' key/cert, server_127.0.0.1_cert.pem already exists".

How do I go about diagnosing the problem and correct it?

**Comment by weslambert** [comment]
What is the output of the following?

`ls -la /opt/so/conf/steno/certs`

**Reply by danielbidwell** [reply]
total 36
drwx------ 2 stenographer socore 4096 Mar 11 07:42 ca
-r--r--r-- 1 stenographer socore 1866 Mar 11 07:42 ca_cert.pem
-r-------- 1 stenographer socore 3243 Mar 11 07:42 ca_key.pem
-r--r--r-- 1 stenographer socore 6456 Mar 11 07:42 client_127.0.0.1_client_cert.pem
-r--r----- 1 stenographer socore 3243 Mar 11 07:42 client_127.0.0.1_client_key.pem
lrwxrwxrwx 1 stenographer socore   32 Mar 11 07:42 client_cert.pem -> client_127.0.0.1_client_cert.pem
lrwxrwxrwx 1 stenographer socore   31 Mar 11 07:42 client_key.pem -> client_127.0.0.1_client_key.pem
-r--r--r-- 1 stenographer socore 6441 Mar 11 07:42 server_127.0.0.1_cert.pem
-r-------- 1 stenographer socore 3243 Mar 11 07:42 server_127.0.0.1_key.pem
lrwxrwxrwx 1 stenographer socore   25 Mar 11 07:42 server_cert.pem -> server_127.0.0.1_cert.pem
lrwxrwxrwx 1 stenographer socore   24 Mar 11 07:42 server_key.pem -> server_127.0.0.1_key.pem

**Reply by KimMura** [reply]
Did you manage to solve this issue? I'm currently facing the exact same problem.

**Comment by danielbidwell** [comment]
I have not found a solution yet.  I am hoping that maybe upgrading to
2.3.40 might fix it.

On Fri, 2021-04-23 at 04:56 -0700, SaltedOnion wrote:
> Did you manage to solve this issue? I'm currently facing the exact
> same problem.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.
-- 
Daniel R. Bidwell       |       ***@***.***
Former Sr. Systems Architect   |       former Chief Information Security Officer
Andrews University      |       Information Technology Services
If two always agree, one of them is unnecessary.
Karma is getting what you deserve,
mercy is not getting what you deserve
grace is getting what you do not deserve.
In theory, theory and practice are the same.
In practice, they are not.
